WNPRC_Purchasing/src/client/components/RequestOrderPanel.tsx

```diff
@@ -63,7 +65,7 @@ export const RequestOrderPanel: FC<Props> = memo(props => {
 
     return (
         <>
-        { !isAdmin && canUpdate && (
+        { ((!isAdmin && canUpdate) || (hasRequestId && !canUpdate && !isAdmin)) && (
```

Comment:
There are a lot of complicated combinations of these checks. Instead of using so many different variants of the check throughout the code, would it be possible to distill them into more understandable booleans? Like canUpdateQuantity, canUpdateAccountAndVendor, etc? If it doesn't improve clarity, feel free to reject this idea.

Reply:
I agree, it is complicated, I'll go ahead and change it..

Reply:
Made changes so that it provides lil more clarity on the users and their access to the field.